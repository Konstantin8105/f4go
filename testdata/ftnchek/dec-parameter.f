C      IMPLICIT INTEGER(a-z)

      PARAMETER shipmoves=3
      PARAMETER fightermoves=8
      PARAMETER (stdparam = 7)
      INTEGER g2(100)
      INTEGER hits(15),tipe(15),crahit(15),craloc(15)

      BYTE      specal,pass
      BYTE      ab,ac,ad,ao,e,own1,own2,own      !AVOID WORD REFERENCES TO THESE
      INTEGER rlmap(3000)
      LOGICAL      automv,autosave,savedone
 
      COMMON/AB9/ab9,prior,nshprf
      COMMON/ARMTOT/armtot
      end
