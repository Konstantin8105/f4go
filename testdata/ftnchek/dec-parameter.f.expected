COMMENT             	C      IMPLICIT INTEGER(a-z)
NEW_LINE            	

NEW_LINE            	

IDENT               	PARAMETER
IDENT               	shipmoves
=                   	=
INT                 	3
NEW_LINE            	

IDENT               	PARAMETER
IDENT               	fightermoves
=                   	=
INT                 	8
NEW_LINE            	

IDENT               	PARAMETER
(                   	(
IDENT               	stdparam
=                   	=
INT                 	7
)                   	)
NEW_LINE            	

INTEGER             	INTEGER
IDENT               	g2
(                   	(
INT                 	100
)                   	)
NEW_LINE            	

INTEGER             	INTEGER
IDENT               	hits
(                   	(
INT                 	15
)                   	)
,                   	,
IDENT               	tipe
(                   	(
INT                 	15
)                   	)
,                   	,
IDENT               	crahit
(                   	(
INT                 	15
)                   	)
,                   	,
IDENT               	craloc
(                   	(
INT                 	15
)                   	)
NEW_LINE            	

NEW_LINE            	

IDENT               	BYTE
IDENT               	specal
,                   	,
IDENT               	pass
NEW_LINE            	

IDENT               	BYTE
IDENT               	ab
,                   	,
IDENT               	ac
,                   	,
IDENT               	ad
,                   	,
IDENT               	ao
,                   	,
IDENT               	e
,                   	,
IDENT               	own1
,                   	,
IDENT               	own2
,                   	,
IDENT               	own
COMMENT             	!AVOID WORD REFERENCES TO THESE
NEW_LINE            	

INTEGER             	INTEGER
IDENT               	rlmap
(                   	(
INT                 	3000
)                   	)
NEW_LINE            	

IDENT               	LOGICAL
IDENT               	automv
,                   	,
IDENT               	autosave
,                   	,
IDENT               	savedone
NEW_LINE            	

NEW_LINE            	

IDENT               	COMMON
/                   	/
IDENT               	AB9
/                   	/
IDENT               	ab9
,                   	,
IDENT               	prior
,                   	,
IDENT               	nshprf
NEW_LINE            	

IDENT               	COMMON
/                   	/
IDENT               	ARMTOT
/                   	/
IDENT               	armtot
NEW_LINE            	

END                 	end
NEW_LINE            	

